pandas/tests/io/test_sql.py

```diff
@@ -3089,6 +3538,10 @@ def test_read_sql_dtype_backend(
     expected = dtype_backend_expected(string_storage, dtype_backend, conn_name)
     tm.assert_frame_equal(result, expected)
 
+    if "adbc" in conn_name:
+        # adbc does not support chunksize argument
+        return
```

Comment:
A `skip` or `xfail` would be preferable especially if we intend on supporting this in the future